packages/framework/src/Resources/config/services_test.yml

```diff
@@ -61,10 +61,20 @@ services:
 
   Shopsys\FrameworkBundle\Component\Domain\Config\DomainsConfigLoader: ~
 
+  Shopsys\FrameworkBundle\Component\Domain\Domain:
```

Comment:
Why is this commit in this branch? It seems like it has nothing to do with the Twig update, am I right? Could you do that in a separate pull request with proper description please?

Reply:
Removed